using_images/sti_images/python.adoc

```diff
@@ -26,27 +33,54 @@ This image comes in two flavors, depending on your needs:
 
 The RHEL 7 image is available through Red Hat's subscription registry via:
 
-****
-`$ docker pull registry.access.redhat.com/openshift/python-33-rhel7`
-****
+----
+$ docker pull registry.access.redhat.com/openshift/python-33-rhel7
+----
 
 *CentOS 7 Based Image*
 
 This image is available on DockerHub. To download it:
 
-****
-`$ docker pull openshift/python-33-centos7`
-****
+----
+$ docker pull openshift/python-33-centos7
+----
 
-To use these images, you can either access them directly from those registries or push them into your OpenShift docker registry.  In addition, it is recommended that you create an ImageStream that points to the image, either in your docker registry or at the external location.  Your OpenShift resources can then reference the ImageStream.  You can find example ImageStream definitions for all the provided OpenShift images https://github.com/openshift/origin/tree/master/examples/image-streams[here].
+To use these images, you can either access them directly from these
+registries, or push them into your OpenShift docker registry. Additionally,
+you can create an ImageStream that points to the image,
+either in your docker registry or at the external location. Your OpenShift
+resources can then reference the ImageStream. You can find
+https://github.com/openshift/origin/tree/master/examples/image-streams[example]
+ImageStream definitions for all the provided OpenShift images.
 
 == Configuration
-The Python image supports a number of environment variables which can be set to control the configuration and behavior of the Python runtime.
-
-To set these environment variables, you can place them into `.sti/environment` file inside your source code repository, or define them in the environment section of the BuildConfig STI Strategy definition.
-
-* [envvar]#APP_FILE# - This variable specifies file name (defaults to `app.py`) passed to the python interpreter which is responsible for launching application.
-
-* [envvar]#APP_MODULE# - This variable specifies WSGI callable. It is of the pattern `$(MODULE_NAME):$(VARIABLE_NAME)`, where module name is a full dotted path and the variable name refers to a inside the specified module. If using `setup.py` for installing the application the module name can be read from that file and variable will default to `application`, eg. see https://github.com/openshift/sti-python/tree/master/3.3/test/setup-test-app[setup-test-app].
-
-* [envvar]#APP_CONFIG# - This variable indicates path to a module which contains http://docs.gunicorn.org/en/latest/configure.html[gunicorn configuration].
+The Python image supports a number of environment variables which can be set to
+control the configuration and behavior of the Python runtime.
+
+To set these environment variables, you can place them into
+`.sti/environment` file inside your source code repository, or define them
```

Comment:
If we're mentioning that option, we should have a link to the documentation explaining how to do this, I'm thinking about [this](http://docs.openshift.org/latest/dev_guide/builds.html#environment-files) place. This also applies to all the other images as well.